restapi/stories.go

```diff
@@ -190,9 +190,29 @@ func (api api) CreateStoriesFormData(w http.ResponseWriter, r *http.Request) ren
 	if err := api.database.Create(&story).Error; err != nil {
 		return rest.ErrInternal(api.logger, err)
 	}
+	names := r.Form["tags"]
+	if len(names) != 0 {
+		err = api.AddTags(names, story)
+		if err != nil {
+			return rest.ErrInvalidRequest(api.logger, "Error Adding Tags", err)
+		}
+	}
 	return rest.MsgStatusOK("Story Added Successfully")
 }
 
+func (api api) AddTags(names []string, story models.Story) error {
+	var length = len(names)
+	tags := make([]models.Tag, length)
+	for i, t := range names {
+		tag := models.Tag{
+			Name:    strings.ToUpper(t),
+			StoryID: story.ID,
+		}
+		tags[i] = tag
```

Comment:
honestly we can probably than just do:
```suggestion
		tags[i] = models.Tag{
			Name:    strings.ToUpper(t),
			StoryID: story.ID,
		}
```